client/my-sites/media/main.jsx

```diff
@@ -192,12 +192,14 @@ class Media extends Component {
 	 * @param  {Function} [callback] - callback function
 	 */
 	deleteMedia( callback ) {
-		const site = this.props.selectedSite;
-		const selected = MediaLibrarySelectedStore.getAll( site.ID );
+		const { selectedSite, translate } = this.props;
+		const selected = MediaLibrarySelectedStore.getAll( selectedSite.ID );
 		const selectedCount = selected.length;
-		const confirmMessage = this.props.translate(
-			'Are you sure you want to permanently delete this item?',
-			'Are you sure you want to permanently delete these items?',
+		const confirmMessage = translate(
+			'Are you sure you want to delete this item? \
```

Comment:
They are *line continuations* and don't really insert *newlines* but I agree that concatenation is less brittle. Changed accordingly.